vm-validator/src/mocks/mock_vm_validator.rs

```diff
@@ -41,8 +41,7 @@ impl TransactionValidation for MockVMValidator {
         };
 
         let sender = txn.sender();
-        let account_dne_test_add =
-            AccountAddress::try_from(&[0 as u8; AccountAddress::LENGTH]).unwrap();
+        let account_dne_test_add = AccountAddress::try_from(&[0 as u8; AccountAddress::LENGTH])?;
```

Comment:
this one... is weird, there are a whole bunch below, but only this one is different, this should really be unified

Reply:
:facepalm:  cleaned that up